_docs_integrate/access-the-connector.md

```diff
@@ -10,16 +10,58 @@ toc: true
 properties:
   - id: SC075
   - category: Getting Started
-  - description: https//enmeshed.eu/integrate/connector-api
+  - description: https://enmeshed.eu/integrate/connector-api
```

Comment:
I would rather not insert any links under 'description', this only happens in about 4 places in the documentation. I would either leave this blank or insert text. (Here and in the Excel)